AlphaWallet/InCoordinator.swift

```diff
@@ -304,6 +304,8 @@ class InCoordinator: NSObject, Coordinator {
     private func setupResourcesOnMultiChain() {
         oneTimeCreationOfOneDatabaseToHoldAllChains()
         setupTokenDataStores()
+        setupNativeCryptoCurrencyPrices()
```

Comment:
Why are these 2 lines needed? They just init the lazy vars, right?

Reply:
we recreate them every time when restart with a new wallet. it needed because of missing rpc server, (when disable or enable some)

Reply:
Oh, because `InCoordinator` is not re-created upon switching servers/wallets. Got it.